components/modal/borrow-more-modal.tsx

```diff
@@ -95,14 +154,35 @@ const BorrowMoreModal = ({
         </div>
         <div className="flex gap-3 justify-between sm:justify-start">
           <div className="text-gray-500">Coupon Purchase Fee</div>
-          <div>24.1234 ETH</div>
+          <div>
+            {formatUnits(
+              interest,
+              position.underlying.decimals,
+              prices[position.underlying.address],
+            )}{' '}
+            {position.underlying.symbol}
+          </div>
         </div>
       </div>
       <button
-        disabled={true}
-        className="font-bold text-base sm:text-xl disabled:bg-gray-100 dark:disabled:bg-gray-800 h-12 sm:h-16 rounded-lg disabled:text-gray-300 dark:disabled:text-gray-500"
+        disabled={
+          position.collateralAmount === 0n ||
+          amount === 0n ||
+          amount > available ||
+          amount + maxInterest > maxLoanAmountExcludingCouponFee
+        }
+        className="font-bold text-base sm:text-xl bg-green-500 disabled:bg-gray-100 dark:disabled:bg-gray-800 h-12 sm:h-16 rounded-lg text-white disabled:text-gray-300 dark:disabled:text-gray-500"
+        onClick={async () => {
+          console.log('borrowing')
+        }}
       >
-        Confirm
+        {amount === 0n
```

Comment:
`Enter loan amount` looks unnecessary.  Let's remove this one.

Reply:
![image](https://github.com/clober-dex/coupon.finance/assets/10525011/d43afbd5-7098-41ea-94d5-ae6061d45fb5)
Nope I think it was needed

Reply:
I was thinking about using custom messages only for error cases, but this looks good too! Let's talk with Luna about what UX is better before launch.

There are three different states.
1. Enabled with Default Message (ex: Confirm)
2. Disabled due to suggestion (ex: Enter loan amount)
3. Disabled due to error state (ex: Insufficient ETH balance)

I was thinking about differentiating between 2 and 3. Let's debate over this in the future. Looks good for now.